state/protocol/protocol_state/epochs/statemachine_test.go

```diff
@@ -140,7 +140,7 @@ func (s *EpochStateMachineSuite) TestBuild_HappyPath() {
 	require.NoError(s.T(), err)
 
 	// prepare a DB update for epoch state
-	s.epochStateDB.On("BatchIndex", rw, s.candidate.ID(), updatedStateID).Return(nil).Once()
+	s.epochStateDB.On("BatchIndex", mocks.Anything, rw, s.candidate.ID(), updatedStateID).Return(nil).Once()
```

Comment:
same here:

1. please update documentation https://github.com/onflow/flow-go/blob/d4865c1fe36941a0f722d9d7f319a70d93d74120/state/protocol/protocol_state/epochs/statemachine_test.go#L151
2. passing a proper instance of `lockctx.Proof` https://github.com/onflow/flow-go/blob/d4865c1fe36941a0f722d9d7f319a70d93d74120/state/protocol/protocol_state/epochs/statemachine_test.go#L154
3. verifying `BatchStore` receives the `lockctx.Proof` https://github.com/onflow/flow-go/blob/d4865c1fe36941a0f722d9d7f319a70d93d74120/state/protocol/protocol_state/epochs/statemachine_test.go#L143